config.yml

```diff
@@ -5,7 +5,7 @@ versions:
   lcov: &lcov_version '1.15-1'
   liboauthcpp: &liboauthcpp_version '0.1.0-1'
   libphonenumber: &libphonenumber_version '8.12.39-1'
-  libpostal: &libpostal_version '1.0.0-1'
+  libpostal: &libpostal_version '1.1-1'
```

Comment:
Uses a non-standard versioning system by leaving off the "patch" version unlike `v.1.0.0` before.